packages/runtime/src/internal/CustomTemplates/expandCustomTemplate.ts

```
@@ -115,45 +116,63 @@ export function expandCustomTemplate<T extends BrickConf | UseSingleBrickConf>(
 
 function expandBrickInTemplate(
-  brickConfInTemplate: BrickConfInTemplate,
-  hostContext: TemplateHostContext
-): RuntimeBrickConfWithTplSymbols {
+  brickConfInTemplate: BrickConf,
+  hostContext: TemplateHostContext,
+  markSlotted?: () => void
+): BrickConf | BrickConf[] {
   // Ignore `if: null` to make `looseCheckIf` working.
   if (brickConfInTemplate.if === null) {
     delete brickConfInTemplate.if;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)